### ISSUE TITLE
[SRAWN-SUBMIT] SR2024 Issue 03

### ISSUE THREAD

**Issue by sedders123** [srobo/srawn#62]
```
sryear: SR2024
issue: 03
submission-date: 2023-09-14T00:00:00Z
publication-date: 2023-09-14T12:00:00Z
```
## SR(A)WN Submission Thread

Student Robotics (Almost) Weekly Newsletter (aka. SR(A)WN) is a regular newsletter published by Student Robotics volunteers for volunteers.

- Keep up with what's going on in SR by subscribing to the SR(A)WN mailing list.
- Let others know about what we've been up to by submitting content to the newsletter.

### Content Submission

Anybody within SR can submit content to the newsletter by commenting on this thread.

The deadline for this issue is midnight UTC on 2023-09-14. Any content submitted after this deadline will be included in the following issue instead.

Please read the submission guidelines before submitting content to SR(A)WN.

<details>
<summary>Submission Guidelines</summary>

- Be inclusive! This is a newsletter for all volunteers, so please try not to assume knowledge (technical, or of specific activities within SR).
- Please follow the [code of conduct](https://opsmanual.studentrobotics.org/about-the-charity/code-of-conduct) when writing and submitting content.
- SR(A)WN is a public newsletter. Please don't submit any sensitive or inappropriate content.
- If additional help is required for something, you can **embolden** it to indicate this to volunteers.
- Please use [Commonmark Markdown](https://commonmark.org/help/) where possible when writing content. Some formatting may be stripped or changed during editing, or by the newsletter template.
- Please use alt text when submitting images to the newsletter, as some mail clients may not be able to read HTML.
</details>

**Comment by RealOrangeOne** [comment]
There was a kit event last weekend to get the kit finalised and ready to be packed into kits. It was a successful and productive event, so thanks to everyone involved.

On the 16th September is the final kit packing event. Here we'll be sorting out the final pieces of kit, ensuring all the software is up-to-date, and packing it into boxes. It's happening at the University of Southampton in Building 32. All volunteers welcome, and no kit experience required.

The kick-off meeting for the V5 iteration of kit will be happening on the 19th September at 7:30. If you're interested in getting involved with what the new vision fro our kit is, please come along - everyone is welcome. This meeting will be very high level, talking about what a new kit might look like, form factor, features, design etc. The meeting link can be found in [our calendar](https://studentrobotics.org/runbook/volunteering/calendars/)

**Comment by RealOrangeOne** [comment]
A huge effort is underway for updating the docs to ensure everything is correct and as approachable as possible for the students. If you're interested in helping out, take a look at the open [PRs](https://github.com/srobo/docs/pulls) or [issues](https://github.com/srobo/docs/issues), or contact the kit team.

**Comment by sedders123** [comment]
## The Game

The SR2024 game has been decided and efforts are underway to write up the rulebook for competitors. A huge thanks to all the volunteers on the Game Special Interest Group who came up with 4 fantastic proposals for us to choose from!

**Comment by RealOrangeOne** [comment]
## Kickstart

Kickstart is coming up soon, and whilst it's a virtual even there's still a lot to be done. If you're interested in helping out, keep an eye on `#competition-team` in Slack or come along to a competition meeting. Team management, rules writing, venue booking, microgames design - plenty to do!

**Comment by RealOrangeOne** [comment]
## V5 kit kick-off meeting

The kick-off meeting for the V5 iteration of kit has been **postponed to 3rd October at 19:15 (UK time)**. If you're interested in getting involved with what the new vision for our kit is, please come along - everyone is welcome. This meeting will be very high level, talking about what a new kit might look like, form factor, features, design etc. The meeting link can be found in [our calendar](https://studentrobotics.org/runbook/volunteering/calendars/)

**Comment by sedders123** [comment]
Closed in #63
